spec/candlepin_scenarios.rb

```diff
@@ -831,20 +831,10 @@ def set_content_access_mode_for_standalone_owner(owner, access_mode, force_distr
   end
 
   def create_owner_in_sca_mode
-    if is_hosted?
-      owner = create_owner(random_string("test_owner"), nil, {
-        'contentAccessModeList' => 'org_environment,entitlement',
-        'contentAccessMode' => "org_environment"
-      })
-    else
-      # Create a new owner that is SCA-capable, but not SCA-enabled
-      owner = create_owner(random_string("test_owner"), nil, {
-          'contentAccessModeList' => 'org_environment,entitlement',
-          'contentAccessMode' => "entitlement"
-      })
-      # and then indirectly make the owner SCA-enabled
-      owner = set_content_access_mode_for_standalone_owner(owner, 'org_environment')
-    end
```

Comment:
Now that we don't have this restriction, we no longer need the `set_content_access_mode_for_standalone_owner` method either, so we can delete it